### DISCUSSION TITLE
German (Deutsch)

### DISCUSSION THREAD

**Discussion by sfr0xyz** [sovenor/bitcoin-rocks#11] [Translations]
Hey! I'd like to contribute to the German translation.
I've started translating `common_de`, `index_de`, `404_de`, `bank-runs_de`, and `compound-inflation-calculator_de`.

To avoid duplicate work, please leave a comment here about which pages you are working on.

Cheers

**Comment by sovenor** [comment]
Amazing, thanks for getting involved!

**Comment by sfr0xyz** [comment]
I've completed the pages listed above (see PR <https://github.com/sovenor/bitcoin-rocks/pull/12>) and will start working on `bitcoin-vs-cbdc_de`, `bitcoin-vs-gold_de`, `get-involved_de`, and `inflation_de`.

**Reply by sovenor** [reply]
It's been merged! Please see my comments on PR #12

**Reply by sfr0xyz** [reply]
Awesome, thank you!
I'll be working on more pages and will open a new PR as soon as I have them finished.

**Reply by sfr0xyz** [reply]
I've noticed that the result message for the compound inflation calculator is not properly displaying the translated strings.
I opened an issue for it: <https://github.com/sovenor/bitcoin-rocks/issues/16>

**Comment by sfr0xyz** [comment]
I've completed the translations of `lightning_de`, `postcards-success_de`, `postcards_de`, `sticker-language-success_de`, `sticker-success_de`, `sticker_de`, `wallets_de`, and `nostr/*`.

I'm going to start working on `sticker-files/*`.

**Comment by sovenor** [comment]
Please note I added a new step to the Translations guide that should be applied to each file:

At the top of the file, change the "last-updated" date to today's date in YYYY-MM-DD format. Change the "locale" to to the [2 letter language code](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes) that corresponds to your language. You can also optionally add your name in the list of authors.

Apologies for the oversight! Please make this change on all of the json files going forward. I am happy to make the change in the pages you already translated.

**Comment by sovenor** [comment]
Hi, please note that there are 4 new translation strings that are now in "common_de.json"

"common_self_custody": "SELF-CUSTODY",
"common_not_your_keys": "NOT YOUR KEYS",
"common_cold_wallet": "COLD WALLET",
"common_hot_wallet": "HOT WALLET"

You should see these at the end of the file when you pull the latest changes. Thanks!

**Reply by sfr0xyz** [reply]
Done in #31

**Comment by sovenor** [comment]
A new translation string has been added to index_de.json:

"home_link_title_energy_6"

Please translate when you have a moment :)

**Reply by sfr0xyz** [reply]
Done in #35

**Comment by sovenor** [comment]
Two new translation strings have been added to the top of common_de.json:

"common_footer_mission_1"
"common_footer_mission_2"

Please translate these when you are able. Thank you!

**Reply by sfr0xyz** [reply]
Done in #44

**Comment by sovenor** [comment]
New translation strings have been added to inflation_de.json:

Line 11: "inflation_sign_got_inflation"
Line 12: "inflation_save_in_bitcoin"
Line 42: "inflation_usd_but_why"

There are also 2 new pages that don't necessarily need translation since the signs campaign is USA only:

signs_de.json
sign-success_de.json

Thanks for your continued contributions!

**Reply by sfr0xyz** [reply]
Done in #46

**Comment by sovenor** [comment]
Translation strings have been modified slightly in wallets_de.json:

Line 11: "two simple questions" is now "these simple questions"
Line 13: "two simple questions" is now "these simple questions"

These new strings have also been added to the same file:

Lines 33 - 39 to add a section about recovery phrases / seed phrases.

Please translate these if you have some spare time. Thank you!
